suite-common/suite-sync/src/storage/tests/refreshSuiteSyncKeys.test.ts

```diff
@@ -0,0 +1,185 @@
+import { EnsureDelegatedIdentityKey } from '@suite-common/delegated-identity-key-types';
+import { EnsureSuiteSyncOwner } from '@suite-common/suite-sync-types';
+import {
+    TrezorDevice,
+    asDelegatedIdentityKey,
+    asSuiteSyncOwnerId,
+    asSuiteSyncOwnerSecretHex,
+} from '@suite-common/suite-types';
+import { ok } from '@trezor/type-utils';
+
+import { LoadSuiteSyncOwnerFromState } from '../../owner/loadSuiteSyncOwnerFromState';
+import { RefreshSuiteSyncKeysDeps, createRefreshSuiteSync } from '../../refreshSuiteSyncKeys';
+
+const createDeps = (): jest.Mocked<RefreshSuiteSyncKeysDeps> => ({
+    dispatch: jest.fn(),
+    ensureSuiteSyncOwner: jest.fn<
+        ReturnType<EnsureSuiteSyncOwner>,
+        Parameters<EnsureSuiteSyncOwner>
+    >(),
+    loadSuiteSyncOwnerFromState: jest.fn<
+        ReturnType<LoadSuiteSyncOwnerFromState>,
+        Parameters<LoadSuiteSyncOwnerFromState>
+    >(),
+    ensureDelegatedIdentityKey: jest.fn<
+        ReturnType<EnsureDelegatedIdentityKey>,
+        Parameters<EnsureDelegatedIdentityKey>
+    >(),
+});
+
+const createDevice = (
+    overrides: Partial<TrezorDevice> = {},
+    deviceStateOverrides?: Partial<TrezorDevice["state"]>,
```

Comment:
```suggestion
    deviceStateOverrides: Partial<TrezorDevice["state"]> = {},
```

Shall work as well so then you can ommit the ternal